### ISSUE TITLE
Accept-Language headers seem to be read wrong.

### ISSUE THREAD

**Issue by half-cambodian-hacker-man** [chubin/wttr.in#116]
Hi. When I send a request to wttr.in through my browser, which sends an Accept-Language header of:

```
Accept-Language: en-GB,en;q=0.8,fr-FR;q=0.6,fr;q=0.4,es-ES;q=0.2,es;q=0.2,ko;q=0.2
```

I see the results in French.

English is the first requested language, and the quality value is set to 0.8.

Why am I seeing output in an accepted-language of lower quality?

**Comment by half-cambodian-hacker-man** [comment]
(The geolocation resolves to where I live in the UK, too.)

**Comment by rwhorman** [comment]
fwiw, I experience the exact same thing ... although from Canada 😉 

```
Accept:text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Accept-Encoding:gzip, deflate
Accept-Language:en-US,en;q=0.8,fr;q=0.6
Cache-Control:no-cache
Connection:keep-alive
Host:wttr.in
Pragma:no-cache
Upgrade-Insecure-Requests:1
User-Agent:Mozilla/5.0 (iPad; CPU OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1
```

**Comment by chubin** [comment]
It's definitively a bug

**Comment by chubin** [comment]
The problem was fixed by @talyian in #304. Please test, if it works for you know, but it definitely should.

@half-cambodian-hacker-man Anthony, in your case `en-GB` (i.e. `en`) should be chosen, but not because it is first, rather because its weight is 1 (if `q=` is not specified, as in the case of `en-GB`, it defaults to 1).
